scripts/dev/clang-format.sh

```diff
@@ -81,7 +81,7 @@ if ! clang-format --version 2>&1 | grep -qPi "^clang-format version $CLANG_FORMA
     err "Need clang-format version $CLANG_FORMAT_VERSION or higher"
 fi
 
-if [[ ! -f lib/kvdb/meson.build ]]; then
+if [[ ! -f meson_options.txt ]]; then
     err "Running from a directory that doesn't seem to be the top of an HSE source tree"
 fi
```

Comment:
`meson_options.txt` is not a good signature for HSE repos.

Reply:
The purpose seems to be to make sure the script is run from the root of the tree, so I am confused why this would not be a better check.

Reply:
My thought was to limit it to HSE repos, and since many repos will have a `meson_options.txt` file it would be better to look for something specific to HSE.  But the script is generic enough I suppose we don't need to enforce that it is an HSE repo.